### ISSUE TITLE
Items in inventory dont show stack count

### ISSUE THREAD

**Issue by IamVelcroboy** [space-wizards/space-station-14#25237]
## Description
When items inside inventory are stacked it doesn't show how much is in the stack. You have to remove the item into your hand, (where the tooltip-count is then obscured by the inventory window.)(Separate issue already opened.)

Ideally, you would be able to see how many of an item you have without having to take it out.  

**Reproduction**
Put stacked items in inventory. Try to determine how many you have.

**Screenshots**
![image](https://github.com/space-wizards/space-station-14/assets/107660393/17d12b14-90bc-4df4-8488-36ef0496ad85)
![image](https://github.com/space-wizards/space-station-14/assets/107660393/05cab8ed-6907-437b-bfc6-277aa964ccb3)
![image](https://github.com/space-wizards/space-station-14/assets/107660393/d5a5eec9-51b0-4d91-83bc-4c5a22b89d6b)
![image](https://github.com/space-wizards/space-station-14/assets/107660393/0d4fbeb6-1dd8-416b-972b-e1ae010d00fa)

**Additional context**
Yes, you can `right-click->Examine->Basic` but that's just too cumbersome imo. Would be ideal if it showed in the inventory itself or the hover-context.

**Comment by 0x6273** [comment]
You can just shift click

**Comment by IamVelcroboy** [comment]
> You can just shift click

Additional context
Yes, you can right-click->Examine->Basic but that's just too cumbersome imo. Would be ideal if it showed in the inventory itself or the hover-context.

**Comment by IamVelcroboy** [comment]
Sorry, I misread a little. Yes, shift+click is slightly faster. I still feel like it should be in the hover context or next to the item.

**Comment by 0x6273** [comment]
Next to the item makes more sense. Hovering and waiting for the hover text is not much faster than shift clicking.

**Comment by IamVelcroboy** [comment]
Yeah, agreed. A little `x123` in the window for stacked items would be sweet QoL.
